src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background: #fff;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
+  order: 1;
```

Comment:
redundant

Reply:
Sorry, but what do you mean under "redundant"? Could you give me a hint of what I am supposed to do?

Reply:
You should remove `order` property